spec/mailers/activity_mailer_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe ActivityMailer, type: :mailer do
-  let(:template_path) { File.join(__dir__, '../fixtures/activity_mailer/test_email') }
+  let(:template_path) { '../../spec/fixtures/activity_mailer/test_email' }
```

Comment:
which means you don't know if it's the correct path. some other global state can affect path resolution. your test will nondeterministically fail.